modules/snapshot/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "snapshot_result" {
+  value = jsondecode(aws_lambda_invocation.snapshot.result)["snaphot"]
```

Comment:
Does this really work? This implies that the Lambda's printing JSON with `snapshot` as a key as its output:

https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseSyntax

It would be cool if this works but without the appropriate JSON log it'll fail. Not sure how `Payload` works there, e.g., is stderr included in payload or no?

(Also, typo)

Reply:
removed the output for now. i'll have to dig into this to get it working. I think we should store the result imo

Reply:
Yeah it's a cool idea to expose this but the first step is for the Lambda itself to produce usable structured output. AFAICT this isn't a thing yet.